metpy/interpolate/tests/test_interpolate_tools.py

```diff
@@ -126,3 +128,14 @@ def test_cressman_weights():
              0.219512195121951]
 
     assert_array_almost_equal(truth, weights)
+
+
+def test_interpolate_to_grid_pandas():
+    r"""Test whether this accepts a pd.Series without crashing."""
```

Comment:
The linter also doesn't like this one line docstring with multiple periods (I think), you could try a change from `pd.Series` to `pandas DataFrame Series`

Reply:
Try putting pd.Series within "`" (backticks).